source/extensions/filters/common/rbac/engine_impl.cc

```diff
@@ -68,7 +85,8 @@ bool RoleBasedAccessControlEngineImpl::checkPolicyMatch(
   bool matched = false;
 
   for (const auto& policy : policies_) {
-    if (policy.second->matches(connection, headers, info)) {
+    if (policy.second->matches(connection, headers, info,
+                               (custom_library_) ? custom_library_.get() : nullptr)) {
```

Comment:
Same as above, I think the simpler version works:
```suggestion
                               custom_library_.get())) {
```

Reply:
ok